src/sdk/pycli/nnicli/nni_client.py

```diff
@@ -129,10 +130,20 @@ def get_job_statistics():
     """return trial job statistics information as a dict"""
     return _nni_rest_get(JOB_STATISTICS_PATH)
 
-def get_job_metrics(trial_job_id=None):
+def get_job_metrics(trial_job_id=None, sort=None):
```

Comment:
`nni_client.py` is only a wrapper of rest api, this is not a good place to add additional features to rest api.
Suggest to:
1.  discuss with team whether to add sort feature into the datastore API.
OR
2.  let end user of nnicli to do the sorting.